modules/oadp-about-backing-and-restoring-from-cluster-to-cluster.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: CONCEPT
+[id="oadp-about-backing-and-restoring-from-cluster-to-cluster_{context}"]
+= About backing up data from one cluster and restoring it on another cluster
+
+{oadp-first} is designed to back up and restore application data in the same {product-title} cluster. {mtc-full} ({mtc-short}) is designed to migrate data from one {product-title} cluster to another or to migrate data from a VMware cluster to an {product-title} cluster.
```

Comment:
Feedback [from slack](https://redhat-internal.slack.com/archives/CFN0XJEPR/p1687983861867299?thread_ts=1687983675.343619&cid=CFN0XJEPR) is that _migrate data from a VMware cluster_ claim is for [MTV](https://catalog.redhat.com/software/container-stacks/detail/60a6e9fe12f690ed33bad15f), not MTC.

Reply:
Right.